pattern_import_export/models/pattern_config.py

```diff
@@ -237,17 +237,21 @@ def _create_pattern_file_export(self, attachment_datas):
         """
         self.ensure_one()
         name = "{name}.{format}".format(name=self.name, format=self.export_format)
-        return self.env["pattern.file"].create(
-            {
-                "name": name,
-                "type": "binary",
-                "res_id": self.id,
-                "res_model": "pattern.config",
-                "datas": attachment_datas,
-                "kind": "export",
-                "state": "done",
-                "pattern_config_id": self.id,
-            }
+        return (
+            self.env["pattern.file"]
```

Comment:
We should not add a sudo here. Creation should be done without sudo.
In think the issue is on computed field that should be run with "compute_sudo=True"